varsim.py

```diff
@@ -261,6 +260,8 @@ def run_randdgv(dgv_file, out_vcf_fd, log_file_fd, seed, sex, options, reference
                         "-ref", os.path.realpath(reference),
                         "-ins", os.path.realpath(insert_seq_file),
                         "-dgv", os.path.realpath(dgv_file)]
+    if len(options.output_all.strip()) > 0:
```

Comment:
Can just use `if options.output_all.strip():`.

Reply:
Ah, good to know, thanks!